scripts/start/run.R

```diff
@@ -207,6 +207,9 @@ run <- function() {
       }
       message("In ", RData_file, ", use current fulldata.gdx path for ", paste(needfulldatagdx, collapse = ", "), ".")
       cfg$files2export$start[needfulldatagdx] <- fulldatapath
+      # let the subsequent run use the renv.lock of this run
+      message("In ", RData_file, ", use current renv.lock for subsequent run ", run, ".")
+      cfg$renvLockFromPreceedingRun <- file.path(cfg_main$remind_folder,cfg_main$results_folder,"renv.lock")
```

Comment:
you might need to add that to `remindextras` in `scripts/start/checkFixCfg.R`

Reply:
Done